src/Octoshift/Services/EnvironmentVariableProvider.cs

```diff
@@ -56,16 +57,27 @@ public EnvironmentVariableProvider(OctoLogger logger)
     public virtual string SmbPassword(bool throwIfNotFound = true) =>
         GetSecret(SMB_PASSWORD, throwIfNotFound);
 
-    private string GetSecret(string secretName, bool throwIfNotFound)
+    public virtual string SkipStatusCheck(bool throwIfNotFound = false) =>
+        GetValue(GEI_SKIP_STATUS_CHECK, throwIfNotFound);
+
+    private string GetValue(string name, bool throwIfNotFound)
     {
-        var secret = Environment.GetEnvironmentVariable(secretName);
+        var value = Environment.GetEnvironmentVariable(name);
 
-        if (string.IsNullOrEmpty(secret))
+#pragma warning disable IDE0046 // Convert to conditional expression
```

Comment:
I ran `dotnet format` and I also didn't like the (so called) simplified version.